spec/requests/splash_screen_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe 'Splash Screen', type: :request do
+  describe 'GET /splash_screen ' do
+    let(:user) { create(:user) }
+
+    it 'and user not logged returns http success' do
+      get root_path
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```

Comment:
- [ ] Good job so far, Kindly add unit tests for the `groups_controller` and `entities_controller` so that we test that the routes are working properly.

Reply:
✅ **Resolved**
I have added unit tests for both the `entities_controller` and `groups_controller`